extensions/podman/src/compatibility-mode.ts

```diff
@@ -239,14 +239,18 @@ export class WindowsSocketCompatibility extends SocketCompatibility {
     const dockerDown = !(await this.isDockerRunning());
     if (dockerDown) {
       await extensionApi.window.showInformationMessage(
-        'Start Docker before continuing. To disable Podman compatibility mode, Docker must be started to override the emulated socket. This may require a restart of Podman Desktop.',
+        'Start Docker before continuing. To disable Podman compatibility mode, Docker must be started to override the emulated socket.',
         'OK',
       );
       return;
```

Comment:
I still don't understand the sequence:

- return seems extra to me
- if user starts Docker Desktop then it won't have the socket as the Podman machine has it so restarting the Podman machine is useless

Reply:
When Docker Desktop starts up it automatically overrides the docker socket and that's the only way to "re enable" the socket mode.

Yup, restarting Podman Machine is useless and that's why I removed the comment.

Reply:
This is where I disagree: start Docker Desktop while the Podman machine is still running will not have the Docker socket taken by the Docker machine because it's already taken by the Podman machine: see my scenario where I start with no machines running then start the Podman one then start Docker Desktop, then stop all of them and starting Docker Desktop and Podman machine and this is where we reached the right state

https://user-images.githubusercontent.com/695993/230144640-67fd3d6c-da99-478d-839e-c95e886a0b10.mp4

Reply:
Ah, that's why I was getting confused. 

That's different on my Windows 10 machine. 

Starting Docker Desktop automatically "re-registers" the socket and I have no issues with having podman machine already running.

Are you using WSL2 / anything specific to your machine?

Reply:
Nothing specific WSL2 Podman machines